src/stories/Introduction.stories.mdx

```diff
@@ -226,8 +226,69 @@ number of tabs and much more limitations of content within the headings. Conside
 [TODO]
 
 ## Progress Indicators
-[TODO]
+progress bar indicates that the user's request has been received and the application is making progress toward completing the requested action.
+
+<div class="formsflow-progressbar-container formsflow-progressbar-success" style={{height:"2px"}}>
+    <div class="formsflow-progressbar-count" aria-valuenow="90%" style={{"left":"90%"}}></div>
+     <div class="formsflow-progressbar">
+       <div class="" style={{"width":"90%"}}></div>
+    </div>
+    <span>Success Message</span>
+</div>
+
+
+<div className="subheading">Usage</div>
+  
+
+  ```html dark
+    <div class="formsflow-progressbar-container formsflow-progressbar-success" style={{height:"2px"}}>
```

Comment:
use correct proper tabbing that might be the reason for parts of the code not getting highlighted in the screenshot